src/components/Inflenser/PostCard.js

```diff
@@ -12,6 +12,7 @@ import { useCampaignManager } from '../../hooks/useCampaignManager';
 import { useCampaign } from '../../hooks/useCampaign';
 import { useSharedState } from '../../context/store';
 import { fetchPublication } from '../../hooks/usePublication';
+import { Container } from 'react-bootstrap';
```

Comment:
Please follow this ordering convention:

- external dependencies
- internal dependencies